include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Solver/IDESolver.h

```diff
@@ -197,19 +197,46 @@ class IDESolver
     }
   }
 
-  /**
-   * Returns the V-type result for the given value at the given statement.
-   * TOP values are never returned.
-   */
+  ///
+  /// Returns the L-type result for the given value at the given statement.
+  ///
   [[nodiscard]] virtual l_t resultAt(n_t stmt, d_t value) {
     return valtab.get(stmt, value);
   }
 
-  /**
-   * Returns the resulting environment for the given statement.
-   * The artificial zero value can be automatically stripped.
-   * TOP values are never returned.
-   */
+  ///
+  /// Returns the L-type result at the given statement for the given data-flow
+  /// fact while respecting LLVM's SSA semantics.
+  ///
+  /// An example: when a value is loaded and the location loaded from, here
+  /// variable 'i', is a data-flow fact that holds, then the loaded value '%0'
+  /// will usually be generated and also holds. However, due to the underlying
+  /// theory (and respective implementation) this load instruction causes the
+  /// loaded value to be generated and thus, it will be valid only AFTER the
+  /// load instruction, i.e., at the successor instruction.
+  ///
+  ///   %0 = load i32, i32* %i, align 4
+  ///
+  /// This result accessor function returns the results at the successor
+  /// instruction(s) reflecting that the expression on the left-hand side holds
+  /// if the expression on the right-hand side holds.
+  ///
```

Comment:
```suggestion
  /// Returns the L-type result at the given statement for the given data-flow
  /// fact while respecting LLVM's SSA semantics.
  ///
  /// An example: when a value is loaded and the location loaded from, here
  /// variable 'i', is a data-flow fact that holds, then the loaded value '%0'
  /// will usually be generated and also holds. However, due to the underlying
  /// theory (and respective implementation) this load instruction causes the
  /// loaded value to be generated and thus, it will be valid only AFTER the
  /// load instruction, i.e., at the successor instruction.
  ///
  ///   %0 = load i32, i32* %i, align 4
  ///
  /// This result accessor function returns the results at the successor
  /// instruction(s) reflecting that the expression on the left-hand side holds
  /// if the expression on the right-hand side holds.
```